French: basic_applications (#1162)
trigaten/Learn_Prompting: (docs)/fr/docs/applied_prompting/mc_tutorial/page.mdx

<p></p>

<Info>
Remarquez comment le modèle raisonne à travers le problème étape par étape.

> Le terme spécifique pour ce comportement est Chaîne de Pensée (@wei2022chain); le modèle génère séquentiellement des déclarations pour arriver à une réponse. Cela est similaire au concept de pensée de Système 2 (issu de [Thinking Fast and Slow](https://en.wikipedia.org/wiki/Thinking,_Fast_and_Slow)); le modèle utilise par défaut la pensée de système 1, mais peut enchaîner la pensée de système 1 pour arriver à une réponse plus méthodique.

Author: ```suggestion
Le terme spécifique pour ce comportement est Chain of Thought (Chaîne de Pensée) (@wei2022chain); le modèle génère séquentiellement des déclarations pour arriver à une réponse. Cela est similaire au concept de pensée de Système 2 (issu de [Thinking Fast and Slow](https://fr.wikipedia.org/wiki/Syst%C3%A8me_1_/_Syst%C3%A8me_2_:_Les_deux_vitesses_de_la_pens%C3%A9e)); le modèle utilise par défaut la pensée de système 1, mais peut enchaîner la pensée de système 1 pour arriver à une réponse plus méthodique.
```

  sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
></iframe>

### Reformuler la question

Rappelez-vous que le prompt original était celui-ci :

```
Laquelle des propositions suivantes, si elle est vraie, renforce le plus l'argument ?
```

Nous pouvons changer le prompt pour celui-ci :

```

> Identifiez chaque choix comme renforçant, affaiblissant ou n'ayant pas d'impact sur l'argument.

Author: to be less wordy in the 3rd category label:

```suggestion
Identifiez chaque choix comme renforçant, affaiblissant ou sans d'impact sur l'argument.
```

  sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
></iframe>

### Ajouter un contexte supplémentaire

Voici un exemple de problème qui peut être facilement résolu en utilisant le théorème de Bayes :

```

> Considérez deux tests médicaux, A et B, pour un virus. Le test A est efficace à 90% pour reconnaître le virus lorsqu'il est présent, mais a un taux de faux positifs de 5% (indiquant que le virus est présent, alors qu'il ne l'est pas). Le test B est efficace à 95% pour reconnaître le virus, mais a un taux de faux positifs de 10%. Les deux tests utilisent des méthodes indépendantes d'identification du virus. Le virus est porté par 2% de la population.

Author: use french punctuation rules: 
```suggestion
Considérez deux tests médicaux, A et B, pour un virus. Le test A est efficace à 90 % pour reconnaître le virus lorsqu'il est présent, mais a un taux de faux positifs de 5 % (indiquant que le virus est présent, alors qu'il ne l'est pas). Le test B est efficace à 95 % pour reconnaître le virus, mais a un taux de faux positifs de 10 %. Les deux tests utilisent des méthodes indépendantes d'identification du virus. Le virus est porté par 2 % de la population.
```
